app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -90,6 +90,72 @@
   }
 }
 
+.user-menu.revamped {
+  right: 0;
+  width: 320px;
+  padding: 0;
+
+  .panel-body-bottom {
+    flex: 0;
+  }
+
+  .menu-tabs-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-left: 1px solid var(--primary-low);
+  }
+
+  .tabs-list {
+    display: flex;
+    flex-direction: column;
+
+    .btn {
+      display: flex;
+      padding: 0.857em;
+      position: relative;
+
+      .d-icon {
+        color: var(--primary-medium);
+      }
+
+      .badge-notification {
+        background-color: var(--tertiary-med-or-tertiary);
+        position: absolute;
+        right: 6px;
+        top: 6px;
+        font-size: var(--font-down-3);
+      }
+
+      &.active {
+        background-color: var(--tertiary-low);
+      }
+      &:hover {
+        background-color: var(--highlight-medium);
+      }
+    }
+  }
+
+  .panel-body-contents {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .quick-access-panel {
+    width: 320px;
+    padding: 0.75em 0.75em 0.75em 0.75em;
+    justify-content: space-between;
+    box-sizing: border-box;
+
+    li {
```

Comment:
this will have a lot of specificity and be hard to customise in the future, could we just use classes please?

Reply:
I've removed the `li` and the `span`s nested inside this rule.